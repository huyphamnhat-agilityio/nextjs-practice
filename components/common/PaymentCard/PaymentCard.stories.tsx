import type { Meta, StoryObj } from "@storybook/react";

// Components
import PaymentCard from ".";

const meta: Meta<typeof PaymentCard> = {
  title: "Components/common/PaymentCard",
  component: PaymentCard,
  argTypes: {
    isNew: {
      description: "Determines whether the payment card is new or not",
    },
  },
  args: {},
  parameters: {
    controls: {
      expanded: true,
    },
  },
};

export default meta;

type Story = StoryObj<typeof PaymentCard>;

export const Primary: Story = {};

export const New: Story = {
  args: {
    isNew: true,
  },
};
