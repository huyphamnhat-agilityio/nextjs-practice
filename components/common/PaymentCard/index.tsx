import { Card, CardBody, CardFooter, CardHeader } from "@nextui-org/react";

// Components
import { CheckMarkIcon, HeartIcon } from "../Icons";
import { Button } from "../Button";

export type PaymentCardProps = {
  isNew?: boolean;
};
const PaymentCard = ({ isNew }: PaymentCardProps) => (
  <Card className="max-w-82 px-10 py-12.5 rounded-none flex flex-col gap-8.75 relative overflow-visible">
    {isNew && (
      <div className="rounded-full bg-dull-orange px-4 py-7 text-white text-2xl font-bold absolute -top-8 -right-8 z-10">
        New
      </div>
    )}
    <CardHeader className="p-0 flex flex-col gap-8.75 text-start items-start">
      <div className="bg-primary p-6 rounded-full">
        <HeartIcon />
      </div>
      <h3 className="uppercase text-foreground text-2xl font-bold">Free</h3>
    </CardHeader>
    <CardBody className="flex flex-col gap-8.75 p-0 text-start items-start">
      <p className="text-foreground-100 text-sm/6 font-bold pr-2">
        Organize across all apps by hand
      </p>

      <div className="flex flex-col items-center">
        <p className="text-primary text-5xl/12.5 font-bold">19$</p>
        <p className="text-secondary-50 text-sm/6 font-bold">Per Month</p>
      </div>

      <p className="text-foreground-100 text-sm">
        Slate helps you see how many more days you need...
      </p>

      <Button size="lg" className="text-white text-sm/5.5 font-bold w-full">
        Try for free
      </Button>
    </CardBody>
    <CardFooter className="flex flex-col items-start gap-3.75 p-0">
      <div className="flex items-center gap-2.5">
        <div className="rounded-full bg-success px-2 py-2.5">
          <CheckMarkIcon />
        </div>
        <p className="text-foreground text-sm/6 font-bold">
          Unlimited product updates
        </p>
      </div>
      <div className="flex items-center gap-2.5">
        <div className="rounded-full bg-success px-2 py-2.5">
          <CheckMarkIcon />
        </div>
        <p className="text-foreground text-sm/6 font-bold">
          Unlimited product updates
        </p>
      </div>
      <div className="flex items-center gap-2.5">
        <div className="rounded-full bg-success px-2 py-2.5">
          <CheckMarkIcon />
        </div>
        <p className="text-foreground text-sm/6 font-bold">
          Unlimited product updates
        </p>
      </div>
      <div className="flex items-center gap-2.5">
        <div className="rounded-full bg-foreground-50 px-2 py-2.5">
          <CheckMarkIcon />
        </div>
        <p className="text-foreground text-sm/6 font-bold">1GB Cloud storage</p>
      </div>
      <div className="flex items-center gap-2.5">
        <div className="rounded-full bg-foreground-50 px-2 py-2.5">
          <CheckMarkIcon />
        </div>
        <p className="text-foreground text-sm/6 font-bold">
          Email and community support
        </p>
      </div>
    </CardFooter>
  </Card>
);

export default PaymentCard;
