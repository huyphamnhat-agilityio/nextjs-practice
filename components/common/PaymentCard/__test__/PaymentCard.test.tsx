import { render } from "@testing-library/react";

// Components
import PaymentCard, { PaymentCardProps } from "..";

describe("PaymentCard test cases", () => {
  const setup = (props?: PaymentCardProps) =>
    render(<PaymentCard {...props} />);

  it("should render correctly", () => {
    const { container } = setup();

    expect(container).toMatchSnapshot();
  });

  it("should render correctly with new badge", () => {
    const { container } = setup({ isNew: true });

    expect(container).toMatchSnapshot();
  });
});
