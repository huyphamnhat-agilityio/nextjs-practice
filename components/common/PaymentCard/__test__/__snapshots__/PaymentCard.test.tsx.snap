// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaymentCard test cases should render correctly 1`] = `
<div>
  <div
    class="h-auto text-foreground box-border bg-content1 outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 shadow-medium transition-transform-background motion-reduce:transition-none max-w-82 px-10 py-12.5 rounded-none flex flex-col gap-8.75 relative overflow-visible"
    tabindex="-1"
  >
    <div
      class="z-10 w-full justify-start shrink-0 overflow-inherit color-inherit subpixel-antialiased rounded-t-large p-0 flex flex-col gap-8.75 text-start items-start"
    >
      <button
        class="z-0 group relative inline-flex items-center justify-center box-border appearance-none select-none whitespace-nowrap font-normal subpixel-antialiased overflow-hidden tap-highlight-transparent data-[pressed=true]:scale-[0.97] outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 text-small gap-2 !gap-0 transition-transform-colors-opacity motion-reduce:transition-none bg-primary text-primary-foreground data-[hover=true]:opacity-hover min-w-0 w-auto h-auto p-6 rounded-full"
        type="button"
      >
        <svg
          fill="none"
          height="48"
          viewBox="0 0 48 48"
          width="48"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M4 16.7999C3.9995 13.9006 5.16477 11.1228 7.23363 9.09156C9.30248 7.06032 12.1012 5.94621 15 5.9999C18.4346 5.98166 21.712 7.43825 24 9.9999C26.288 7.43825 29.5654 5.98166 33 5.9999C35.8988 5.94621 38.6975 7.06032 40.7664 9.09156C42.8352 11.1228 44.0005 13.9006 44 16.7999C44 27.5119 31.242 35.5999 24 41.9999C16.774 35.5459 4 27.5199 4 16.7999Z"
            fill="white"
          />
        </svg>
      </button>
      <h3
        class="uppercase text-foreground text-2xl font-bold"
      >
        Free
      </h3>
    </div>
    <div
      class="relative w-full flex-auto place-content-inherit align-items-inherit h-auto break-words overflow-y-auto subpixel-antialiased flex flex-col gap-8.75 p-0 text-start items-start"
    >
      <h6
        class="text-foreground-100 text-sm/6 font-bold pr-2"
      >
        Organize across all apps by hand
      </h6>
      <div
        class="flex flex-col items-center"
      >
        <p
          class="text-primary text-5xl/12.5 font-bold"
        >
          19$
        </p>
        <p
          class="text-secondary-50 text-sm/6 font-bold"
        >
          Per Month
        </p>
      </div>
      <p
        class="text-foreground-100 text-sm"
      >
        Slate helps you see how many more days you need...
      </p>
      <button
        class="z-0 group relative inline-flex items-center justify-center box-border appearance-none select-none whitespace-nowrap subpixel-antialiased overflow-hidden tap-highlight-transparent data-[pressed=true]:scale-[0.97] outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 min-w-24 gap-3 rounded-large [&>svg]:max-w-[theme(spacing.8)] transition-transform-colors-opacity motion-reduce:transition-none bg-primary data-[hover=true]:opacity-hover px-10 py-3.75 h-auto rounded-1.25 text-white text-sm/5.5 font-bold w-full"
        type="button"
      >
        Try for free
      </button>
    </div>
    <div
      class="h-auto w-full overflow-hidden color-inherit subpixel-antialiased rounded-b-large flex flex-col items-start gap-3.75 p-0"
    >
      <div
        class="flex items-center gap-2.5"
      >
        <div
          class="rounded-full bg-success px-2 py-2.5"
        >
          <svg
            fill="none"
            height="11"
            viewBox="0 0 16 11"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              clip-path="url(#clip0_2685_1156)"
            >
              <path
                clip-rule="evenodd"
                d="M16.2957 0.204347C16.3604 0.268958 16.4118 0.345714 16.4469 0.430218C16.482 0.514722 16.5 0.605313 16.5 0.696803C16.5 0.788294 16.482 0.878885 16.4469 0.963389C16.4118 1.04789 16.3604 1.12465 16.2957 1.18926L6.55781 10.9271C6.4932 10.9919 6.41645 11.0433 6.33194 11.0783C6.24744 11.1134 6.15685 11.1314 6.06536 11.1314C5.97387 11.1314 5.88328 11.1134 5.79877 11.0783C5.71427 11.0433 5.63751 10.9919 5.5729 10.9271L0.703982 6.05818C0.573374 5.92757 0.5 5.75043 0.5 5.56572C0.5 5.38102 0.573374 5.20387 0.703982 5.07327C0.83459 4.94266 1.01173 4.86928 1.19644 4.86928C1.38115 4.86928 1.55829 4.94266 1.6889 5.07327L6.06536 9.45112L15.3107 0.204347C15.3754 0.139572 15.4521 0.08818 15.5366 0.0531148C15.6211 0.0180496 15.7117 0 15.8032 0C15.8947 0 15.9853 0.0180496 16.0698 0.0531148C16.1543 0.08818 16.231 0.139572 16.2957 0.204347Z"
                fill="white"
                fill-rule="evenodd"
              />
            </g>
            <defs>
              <clippath
                id="clip0_2685_1156"
              >
                <rect
                  fill="white"
                  height="11"
                  width="16"
                />
              </clippath>
            </defs>
          </svg>
        </div>
        <p
          class="text-foreground text-sm/6 font-bold"
        >
          Unlimited product updates
        </p>
      </div>
      <div
        class="flex items-center gap-2.5"
      >
        <div
          class="rounded-full bg-success px-2 py-2.5"
        >
          <svg
            fill="none"
            height="11"
            viewBox="0 0 16 11"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              clip-path="url(#clip0_2685_1156)"
            >
              <path
                clip-rule="evenodd"
                d="M16.2957 0.204347C16.3604 0.268958 16.4118 0.345714 16.4469 0.430218C16.482 0.514722 16.5 0.605313 16.5 0.696803C16.5 0.788294 16.482 0.878885 16.4469 0.963389C16.4118 1.04789 16.3604 1.12465 16.2957 1.18926L6.55781 10.9271C6.4932 10.9919 6.41645 11.0433 6.33194 11.0783C6.24744 11.1134 6.15685 11.1314 6.06536 11.1314C5.97387 11.1314 5.88328 11.1134 5.79877 11.0783C5.71427 11.0433 5.63751 10.9919 5.5729 10.9271L0.703982 6.05818C0.573374 5.92757 0.5 5.75043 0.5 5.56572C0.5 5.38102 0.573374 5.20387 0.703982 5.07327C0.83459 4.94266 1.01173 4.86928 1.19644 4.86928C1.38115 4.86928 1.55829 4.94266 1.6889 5.07327L6.06536 9.45112L15.3107 0.204347C15.3754 0.139572 15.4521 0.08818 15.5366 0.0531148C15.6211 0.0180496 15.7117 0 15.8032 0C15.8947 0 15.9853 0.0180496 16.0698 0.0531148C16.1543 0.08818 16.231 0.139572 16.2957 0.204347Z"
                fill="white"
                fill-rule="evenodd"
              />
            </g>
            <defs>
              <clippath
                id="clip0_2685_1156"
              >
                <rect
                  fill="white"
                  height="11"
                  width="16"
                />
              </clippath>
            </defs>
          </svg>
        </div>
        <p
          class="text-foreground text-sm/6 font-bold"
        >
          Unlimited product updates
        </p>
      </div>
      <div
        class="flex items-center gap-2.5"
      >
        <div
          class="rounded-full bg-success px-2 py-2.5"
        >
          <svg
            fill="none"
            height="11"
            viewBox="0 0 16 11"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              clip-path="url(#clip0_2685_1156)"
            >
              <path
                clip-rule="evenodd"
                d="M16.2957 0.204347C16.3604 0.268958 16.4118 0.345714 16.4469 0.430218C16.482 0.514722 16.5 0.605313 16.5 0.696803C16.5 0.788294 16.482 0.878885 16.4469 0.963389C16.4118 1.04789 16.3604 1.12465 16.2957 1.18926L6.55781 10.9271C6.4932 10.9919 6.41645 11.0433 6.33194 11.0783C6.24744 11.1134 6.15685 11.1314 6.06536 11.1314C5.97387 11.1314 5.88328 11.1134 5.79877 11.0783C5.71427 11.0433 5.63751 10.9919 5.5729 10.9271L0.703982 6.05818C0.573374 5.92757 0.5 5.75043 0.5 5.56572C0.5 5.38102 0.573374 5.20387 0.703982 5.07327C0.83459 4.94266 1.01173 4.86928 1.19644 4.86928C1.38115 4.86928 1.55829 4.94266 1.6889 5.07327L6.06536 9.45112L15.3107 0.204347C15.3754 0.139572 15.4521 0.08818 15.5366 0.0531148C15.6211 0.0180496 15.7117 0 15.8032 0C15.8947 0 15.9853 0.0180496 16.0698 0.0531148C16.1543 0.08818 16.231 0.139572 16.2957 0.204347Z"
                fill="white"
                fill-rule="evenodd"
              />
            </g>
            <defs>
              <clippath
                id="clip0_2685_1156"
              >
                <rect
                  fill="white"
                  height="11"
                  width="16"
                />
              </clippath>
            </defs>
          </svg>
        </div>
        <p
          class="text-foreground text-sm/6 font-bold"
        >
          Unlimited product updates
        </p>
      </div>
      <div
        class="flex items-center gap-2.5"
      >
        <div
          class="rounded-full bg-foreground-50 px-2 py-2.5"
        >
          <svg
            fill="none"
            height="11"
            viewBox="0 0 16 11"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              clip-path="url(#clip0_2685_1156)"
            >
              <path
                clip-rule="evenodd"
                d="M16.2957 0.204347C16.3604 0.268958 16.4118 0.345714 16.4469 0.430218C16.482 0.514722 16.5 0.605313 16.5 0.696803C16.5 0.788294 16.482 0.878885 16.4469 0.963389C16.4118 1.04789 16.3604 1.12465 16.2957 1.18926L6.55781 10.9271C6.4932 10.9919 6.41645 11.0433 6.33194 11.0783C6.24744 11.1134 6.15685 11.1314 6.06536 11.1314C5.97387 11.1314 5.88328 11.1134 5.79877 11.0783C5.71427 11.0433 5.63751 10.9919 5.5729 10.9271L0.703982 6.05818C0.573374 5.92757 0.5 5.75043 0.5 5.56572C0.5 5.38102 0.573374 5.20387 0.703982 5.07327C0.83459 4.94266 1.01173 4.86928 1.19644 4.86928C1.38115 4.86928 1.55829 4.94266 1.6889 5.07327L6.06536 9.45112L15.3107 0.204347C15.3754 0.139572 15.4521 0.08818 15.5366 0.0531148C15.6211 0.0180496 15.7117 0 15.8032 0C15.8947 0 15.9853 0.0180496 16.0698 0.0531148C16.1543 0.08818 16.231 0.139572 16.2957 0.204347Z"
                fill="white"
                fill-rule="evenodd"
              />
            </g>
            <defs>
              <clippath
                id="clip0_2685_1156"
              >
                <rect
                  fill="white"
                  height="11"
                  width="16"
                />
              </clippath>
            </defs>
          </svg>
        </div>
        <p
          class="text-foreground text-sm/6 font-bold"
        >
          1GB Cloud storage
        </p>
      </div>
      <div
        class="flex items-center gap-2.5"
      >
        <div
          class="rounded-full bg-foreground-50 px-2 py-2.5"
        >
          <svg
            fill="none"
            height="11"
            viewBox="0 0 16 11"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              clip-path="url(#clip0_2685_1156)"
            >
              <path
                clip-rule="evenodd"
                d="M16.2957 0.204347C16.3604 0.268958 16.4118 0.345714 16.4469 0.430218C16.482 0.514722 16.5 0.605313 16.5 0.696803C16.5 0.788294 16.482 0.878885 16.4469 0.963389C16.4118 1.04789 16.3604 1.12465 16.2957 1.18926L6.55781 10.9271C6.4932 10.9919 6.41645 11.0433 6.33194 11.0783C6.24744 11.1134 6.15685 11.1314 6.06536 11.1314C5.97387 11.1314 5.88328 11.1134 5.79877 11.0783C5.71427 11.0433 5.63751 10.9919 5.5729 10.9271L0.703982 6.05818C0.573374 5.92757 0.5 5.75043 0.5 5.56572C0.5 5.38102 0.573374 5.20387 0.703982 5.07327C0.83459 4.94266 1.01173 4.86928 1.19644 4.86928C1.38115 4.86928 1.55829 4.94266 1.6889 5.07327L6.06536 9.45112L15.3107 0.204347C15.3754 0.139572 15.4521 0.08818 15.5366 0.0531148C15.6211 0.0180496 15.7117 0 15.8032 0C15.8947 0 15.9853 0.0180496 16.0698 0.0531148C16.1543 0.08818 16.231 0.139572 16.2957 0.204347Z"
                fill="white"
                fill-rule="evenodd"
              />
            </g>
            <defs>
              <clippath
                id="clip0_2685_1156"
              >
                <rect
                  fill="white"
                  height="11"
                  width="16"
                />
              </clippath>
            </defs>
          </svg>
        </div>
        <p
          class="text-foreground text-sm/6 font-bold"
        >
          Email and community support
        </p>
      </div>
    </div>
  </div>
</div>
`;

exports[`PaymentCard test cases should render correctly with new badge 1`] = `
<div>
  <div
    class="h-auto text-foreground box-border bg-content1 outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 shadow-medium transition-transform-background motion-reduce:transition-none max-w-82 px-10 py-12.5 rounded-none flex flex-col gap-8.75 relative overflow-visible"
    tabindex="-1"
  >
    <div
      class="rounded-full bg-dull-orange px-4 py-7 text-white text-2xl font-bold absolute -top-8 -right-8 z-10"
    >
      New
    </div>
    <div
      class="z-10 w-full justify-start shrink-0 overflow-inherit color-inherit subpixel-antialiased rounded-t-large p-0 flex flex-col gap-8.75 text-start items-start"
    >
      <button
        class="z-0 group relative inline-flex items-center justify-center box-border appearance-none select-none whitespace-nowrap font-normal subpixel-antialiased overflow-hidden tap-highlight-transparent data-[pressed=true]:scale-[0.97] outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 text-small gap-2 !gap-0 transition-transform-colors-opacity motion-reduce:transition-none bg-primary text-primary-foreground data-[hover=true]:opacity-hover min-w-0 w-auto h-auto p-6 rounded-full"
        type="button"
      >
        <svg
          fill="none"
          height="48"
          viewBox="0 0 48 48"
          width="48"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M4 16.7999C3.9995 13.9006 5.16477 11.1228 7.23363 9.09156C9.30248 7.06032 12.1012 5.94621 15 5.9999C18.4346 5.98166 21.712 7.43825 24 9.9999C26.288 7.43825 29.5654 5.98166 33 5.9999C35.8988 5.94621 38.6975 7.06032 40.7664 9.09156C42.8352 11.1228 44.0005 13.9006 44 16.7999C44 27.5119 31.242 35.5999 24 41.9999C16.774 35.5459 4 27.5199 4 16.7999Z"
            fill="white"
          />
        </svg>
      </button>
      <h3
        class="uppercase text-foreground text-2xl font-bold"
      >
        Free
      </h3>
    </div>
    <div
      class="relative w-full flex-auto place-content-inherit align-items-inherit h-auto break-words overflow-y-auto subpixel-antialiased flex flex-col gap-8.75 p-0 text-start items-start"
    >
      <h6
        class="text-foreground-100 text-sm/6 font-bold pr-2"
      >
        Organize across all apps by hand
      </h6>
      <div
        class="flex flex-col items-center"
      >
        <p
          class="text-primary text-5xl/12.5 font-bold"
        >
          19$
        </p>
        <p
          class="text-secondary-50 text-sm/6 font-bold"
        >
          Per Month
        </p>
      </div>
      <p
        class="text-foreground-100 text-sm"
      >
        Slate helps you see how many more days you need...
      </p>
      <button
        class="z-0 group relative inline-flex items-center justify-center box-border appearance-none select-none whitespace-nowrap subpixel-antialiased overflow-hidden tap-highlight-transparent data-[pressed=true]:scale-[0.97] outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 min-w-24 gap-3 rounded-large [&>svg]:max-w-[theme(spacing.8)] transition-transform-colors-opacity motion-reduce:transition-none bg-primary data-[hover=true]:opacity-hover px-10 py-3.75 h-auto rounded-1.25 text-white text-sm/5.5 font-bold w-full"
        type="button"
      >
        Try for free
      </button>
    </div>
    <div
      class="h-auto w-full overflow-hidden color-inherit subpixel-antialiased rounded-b-large flex flex-col items-start gap-3.75 p-0"
    >
      <div
        class="flex items-center gap-2.5"
      >
        <div
          class="rounded-full bg-success px-2 py-2.5"
        >
          <svg
            fill="none"
            height="11"
            viewBox="0 0 16 11"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              clip-path="url(#clip0_2685_1156)"
            >
              <path
                clip-rule="evenodd"
                d="M16.2957 0.204347C16.3604 0.268958 16.4118 0.345714 16.4469 0.430218C16.482 0.514722 16.5 0.605313 16.5 0.696803C16.5 0.788294 16.482 0.878885 16.4469 0.963389C16.4118 1.04789 16.3604 1.12465 16.2957 1.18926L6.55781 10.9271C6.4932 10.9919 6.41645 11.0433 6.33194 11.0783C6.24744 11.1134 6.15685 11.1314 6.06536 11.1314C5.97387 11.1314 5.88328 11.1134 5.79877 11.0783C5.71427 11.0433 5.63751 10.9919 5.5729 10.9271L0.703982 6.05818C0.573374 5.92757 0.5 5.75043 0.5 5.56572C0.5 5.38102 0.573374 5.20387 0.703982 5.07327C0.83459 4.94266 1.01173 4.86928 1.19644 4.86928C1.38115 4.86928 1.55829 4.94266 1.6889 5.07327L6.06536 9.45112L15.3107 0.204347C15.3754 0.139572 15.4521 0.08818 15.5366 0.0531148C15.6211 0.0180496 15.7117 0 15.8032 0C15.8947 0 15.9853 0.0180496 16.0698 0.0531148C16.1543 0.08818 16.231 0.139572 16.2957 0.204347Z"
                fill="white"
                fill-rule="evenodd"
              />
            </g>
            <defs>
              <clippath
                id="clip0_2685_1156"
              >
                <rect
                  fill="white"
                  height="11"
                  width="16"
                />
              </clippath>
            </defs>
          </svg>
        </div>
        <p
          class="text-foreground text-sm/6 font-bold"
        >
          Unlimited product updates
        </p>
      </div>
      <div
        class="flex items-center gap-2.5"
      >
        <div
          class="rounded-full bg-success px-2 py-2.5"
        >
          <svg
            fill="none"
            height="11"
            viewBox="0 0 16 11"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              clip-path="url(#clip0_2685_1156)"
            >
              <path
                clip-rule="evenodd"
                d="M16.2957 0.204347C16.3604 0.268958 16.4118 0.345714 16.4469 0.430218C16.482 0.514722 16.5 0.605313 16.5 0.696803C16.5 0.788294 16.482 0.878885 16.4469 0.963389C16.4118 1.04789 16.3604 1.12465 16.2957 1.18926L6.55781 10.9271C6.4932 10.9919 6.41645 11.0433 6.33194 11.0783C6.24744 11.1134 6.15685 11.1314 6.06536 11.1314C5.97387 11.1314 5.88328 11.1134 5.79877 11.0783C5.71427 11.0433 5.63751 10.9919 5.5729 10.9271L0.703982 6.05818C0.573374 5.92757 0.5 5.75043 0.5 5.56572C0.5 5.38102 0.573374 5.20387 0.703982 5.07327C0.83459 4.94266 1.01173 4.86928 1.19644 4.86928C1.38115 4.86928 1.55829 4.94266 1.6889 5.07327L6.06536 9.45112L15.3107 0.204347C15.3754 0.139572 15.4521 0.08818 15.5366 0.0531148C15.6211 0.0180496 15.7117 0 15.8032 0C15.8947 0 15.9853 0.0180496 16.0698 0.0531148C16.1543 0.08818 16.231 0.139572 16.2957 0.204347Z"
                fill="white"
                fill-rule="evenodd"
              />
            </g>
            <defs>
              <clippath
                id="clip0_2685_1156"
              >
                <rect
                  fill="white"
                  height="11"
                  width="16"
                />
              </clippath>
            </defs>
          </svg>
        </div>
        <p
          class="text-foreground text-sm/6 font-bold"
        >
          Unlimited product updates
        </p>
      </div>
      <div
        class="flex items-center gap-2.5"
      >
        <div
          class="rounded-full bg-success px-2 py-2.5"
        >
          <svg
            fill="none"
            height="11"
            viewBox="0 0 16 11"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              clip-path="url(#clip0_2685_1156)"
            >
              <path
                clip-rule="evenodd"
                d="M16.2957 0.204347C16.3604 0.268958 16.4118 0.345714 16.4469 0.430218C16.482 0.514722 16.5 0.605313 16.5 0.696803C16.5 0.788294 16.482 0.878885 16.4469 0.963389C16.4118 1.04789 16.3604 1.12465 16.2957 1.18926L6.55781 10.9271C6.4932 10.9919 6.41645 11.0433 6.33194 11.0783C6.24744 11.1134 6.15685 11.1314 6.06536 11.1314C5.97387 11.1314 5.88328 11.1134 5.79877 11.0783C5.71427 11.0433 5.63751 10.9919 5.5729 10.9271L0.703982 6.05818C0.573374 5.92757 0.5 5.75043 0.5 5.56572C0.5 5.38102 0.573374 5.20387 0.703982 5.07327C0.83459 4.94266 1.01173 4.86928 1.19644 4.86928C1.38115 4.86928 1.55829 4.94266 1.6889 5.07327L6.06536 9.45112L15.3107 0.204347C15.3754 0.139572 15.4521 0.08818 15.5366 0.0531148C15.6211 0.0180496 15.7117 0 15.8032 0C15.8947 0 15.9853 0.0180496 16.0698 0.0531148C16.1543 0.08818 16.231 0.139572 16.2957 0.204347Z"
                fill="white"
                fill-rule="evenodd"
              />
            </g>
            <defs>
              <clippath
                id="clip0_2685_1156"
              >
                <rect
                  fill="white"
                  height="11"
                  width="16"
                />
              </clippath>
            </defs>
          </svg>
        </div>
        <p
          class="text-foreground text-sm/6 font-bold"
        >
          Unlimited product updates
        </p>
      </div>
      <div
        class="flex items-center gap-2.5"
      >
        <div
          class="rounded-full bg-foreground-50 px-2 py-2.5"
        >
          <svg
            fill="none"
            height="11"
            viewBox="0 0 16 11"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              clip-path="url(#clip0_2685_1156)"
            >
              <path
                clip-rule="evenodd"
                d="M16.2957 0.204347C16.3604 0.268958 16.4118 0.345714 16.4469 0.430218C16.482 0.514722 16.5 0.605313 16.5 0.696803C16.5 0.788294 16.482 0.878885 16.4469 0.963389C16.4118 1.04789 16.3604 1.12465 16.2957 1.18926L6.55781 10.9271C6.4932 10.9919 6.41645 11.0433 6.33194 11.0783C6.24744 11.1134 6.15685 11.1314 6.06536 11.1314C5.97387 11.1314 5.88328 11.1134 5.79877 11.0783C5.71427 11.0433 5.63751 10.9919 5.5729 10.9271L0.703982 6.05818C0.573374 5.92757 0.5 5.75043 0.5 5.56572C0.5 5.38102 0.573374 5.20387 0.703982 5.07327C0.83459 4.94266 1.01173 4.86928 1.19644 4.86928C1.38115 4.86928 1.55829 4.94266 1.6889 5.07327L6.06536 9.45112L15.3107 0.204347C15.3754 0.139572 15.4521 0.08818 15.5366 0.0531148C15.6211 0.0180496 15.7117 0 15.8032 0C15.8947 0 15.9853 0.0180496 16.0698 0.0531148C16.1543 0.08818 16.231 0.139572 16.2957 0.204347Z"
                fill="white"
                fill-rule="evenodd"
              />
            </g>
            <defs>
              <clippath
                id="clip0_2685_1156"
              >
                <rect
                  fill="white"
                  height="11"
                  width="16"
                />
              </clippath>
            </defs>
          </svg>
        </div>
        <p
          class="text-foreground text-sm/6 font-bold"
        >
          1GB Cloud storage
        </p>
      </div>
      <div
        class="flex items-center gap-2.5"
      >
        <div
          class="rounded-full bg-foreground-50 px-2 py-2.5"
        >
          <svg
            fill="none"
            height="11"
            viewBox="0 0 16 11"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              clip-path="url(#clip0_2685_1156)"
            >
              <path
                clip-rule="evenodd"
                d="M16.2957 0.204347C16.3604 0.268958 16.4118 0.345714 16.4469 0.430218C16.482 0.514722 16.5 0.605313 16.5 0.696803C16.5 0.788294 16.482 0.878885 16.4469 0.963389C16.4118 1.04789 16.3604 1.12465 16.2957 1.18926L6.55781 10.9271C6.4932 10.9919 6.41645 11.0433 6.33194 11.0783C6.24744 11.1134 6.15685 11.1314 6.06536 11.1314C5.97387 11.1314 5.88328 11.1134 5.79877 11.0783C5.71427 11.0433 5.63751 10.9919 5.5729 10.9271L0.703982 6.05818C0.573374 5.92757 0.5 5.75043 0.5 5.56572C0.5 5.38102 0.573374 5.20387 0.703982 5.07327C0.83459 4.94266 1.01173 4.86928 1.19644 4.86928C1.38115 4.86928 1.55829 4.94266 1.6889 5.07327L6.06536 9.45112L15.3107 0.204347C15.3754 0.139572 15.4521 0.08818 15.5366 0.0531148C15.6211 0.0180496 15.7117 0 15.8032 0C15.8947 0 15.9853 0.0180496 16.0698 0.0531148C16.1543 0.08818 16.231 0.139572 16.2957 0.204347Z"
                fill="white"
                fill-rule="evenodd"
              />
            </g>
            <defs>
              <clippath
                id="clip0_2685_1156"
              >
                <rect
                  fill="white"
                  height="11"
                  width="16"
                />
              </clippath>
            </defs>
          </svg>
        </div>
        <p
          class="text-foreground text-sm/6 font-bold"
        >
          Email and community support
        </p>
      </div>
    </div>
  </div>
</div>
`;
